tests/integration_tests/test_cli.py

```diff
@@ -84,6 +86,25 @@ def generate_wallet(coldkey : 'Keypair' = None, hotkey: 'Keypair' = None):
 
         return wallet
 
+    @staticmethod
+    def init_wallet():
+        if os.path.exists('/tmp/pytest'):
+            shutil.rmtree('/tmp/pytest')
+    
+        the_wallet = bittensor.wallet (
+            path = '/tmp/pytest',
+            name = 'pytest',
+            hotkey = 'pytest',
+        )
+        
+        return the_wallet
```

Comment:
This method already exists in `integration_tests/test_wallet.py`. Rather than redefining it here, why not move it into something like `tests/utils.py` and import into the tests that need it?

Reply:
Removed in favour of mocking the IO